src/features/hotspots/setup/HotspotSelectionScreen.tsx

```diff
@@ -109,6 +111,89 @@ const HotspotSetupSelectionScreen = () => {
           </TouchableHighlightBox>
         </Card>
       </Box>
+
+      <Box flexDirection="row" height={191} marginTop="s">
```

Comment:
I think we should pull this into another component called `HotspotSelectionCard` or something then you can just pass in the type